const.go

```diff
@@ -8,12 +8,23 @@
 
 package mysql
 
+import "runtime"
+
 const (
 	defaultAuthPlugin       = "mysql_native_password"
 	defaultMaxAllowedPacket = 4 << 20 // 4 MiB
 	minProtocolVersion      = 10
 	maxPacketSize           = 1<<24 - 1
 	timeFormat              = "2006-01-02 15:04:05.999999"
+
+	// Connection attributes
+	// See https://dev.mysql.com/doc/refman/8.0/en/performance-schema-connection-attribute-tables.html#performance-schema-connection-attributes-available
+	connAttrClientName      = "_client_name"
```

Comment:
The `_client_name` is very useful. Would be nice to get the version of the driver. Not sure if there is a good, reliable and easy way to do this.

Reply:
version is useful too. But there isn't a simple way to get it. 
Shall we set a const value here? 
For example, `connAttrVersionValue = "v1.7"`. 
If we use this way, we need to manually update it with each release. It's very inconvenient.

Reply:
Updating is quite error prone. And embedding the git commit id etc doesn't seem to be easily/reliable doable with older versions of Go. Let's not do this for now.